CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 This file is used to list changes made in each version of the htpasswd cookbook.
 
+## Unreleased
+
+### Fixed
+
+ - fix helper script to use file instead of name property
+
```

Comment:
For a simple fix like this, let's have this instead. This should also fix the `mdl` error on the tests.
```suggestion
## Unreleased

- Fix helper script to use file instead of name property

```

Reply:
fixed